compiler/rustc_middle/src/mir/syntax.rs

```diff
@@ -931,6 +931,8 @@ pub enum TerminatorKind<'tcx> {
         asm_macro: InlineAsmMacro,
 
         /// The template for the inline assembly, with placeholders.
+        #[type_foldable(identity)]
```

Comment:
Ideally we'd have most of these MIR fields marked as ignore/identity, but these are the only ones that we need to get rid of some annoying noop impls.